src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import User from './User';
+
+export default interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User,
```

Comment:
```suggestion
  user?: User,
```
user can be undefined